skk.el

```diff
@@ -835,11 +835,13 @@ Delete Selection $B%b!<%I$,(B SKK $B$r;H$C$?F|K\8lF~NO$KBP$7$F$b5!G=$9$k$h$&$
     (setq skk-okuri-nasi-min (point-marker))))
 
 (defun skk-setup-melpa-isearch ()
-  (and (string-match "melpa" (skk-version))
+  (and (string-match-p "melpa" (skk-version))
```

Comment:
`string-match` is changing the match data. Replacing with `string-match-p` avoids side effects.